tslib/react/src/components/PlotIntermediateValues.tsx

```diff
@@ -79,23 +83,23 @@ const plotIntermediateValue = (
         t.values.length > 0) ||
       t.state === "Running"
   )
+
   const plotData: Partial<plotly.PlotData>[] = filteredTrials.map((trial) => {
-    const values = trial.intermediate_values.filter(
-      (iv) =>
-        iv.value !== Infinity &&
-        iv.value !== -Infinity &&
-        !Number.isNaN(iv.value)
-    )
```

Comment:
I removed those lines because they do not exist in optuna-dashbaord. But, if you feel we should revert them, tell me that, please.